spec/changelog.yml

```diff
@@ -4,461 +4,464 @@
 ##
 - version: 3.4.0-next
   changes:
-  - description: Add support for content packages.
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/777
+    - description: Add support for content packages.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/777
 - version: 3.3.0-next
   changes:
-  - description: Add support for `slo` assets.
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/767
+    - description: Add support for `slo` assets.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/767
+    - description: Add support for identity of agentless deployment mode.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/801
```

Comment:
Please put this change in a new `3.2.3-next` version, we add changes to new minors only if they require changes in Kibana for the package to properly work.

Also, maybe we should add a breaking change notice mentioning that we are adding required fields.

Reply:
Is this the only place where we should specify the breaking change?

https://github.com/elastic/package-spec/pull/801/files#diff-2782fabcaeaa32cbab3eb1fabfbae4aa717d626056e383732eb8e014797fadbfR18

Reply:
Yes, this is enough.